crmsh/bootstrap.py

```diff
@@ -1333,15 +1336,33 @@ def configure_qdevice_interactive():
     logger.info("Configure Qdevice/Qnetd:\n" + QDEVICE_HELP_INFO + "\n")
     if not confirm("Do you want to configure QDevice?"):
         return
-    qnetd_addr = prompt_for_string("HOST or IP of the QNetd server to be used")
-    if not qnetd_addr:
-        utils.fatal("Address of QNetd is required")
-    qdevice_port = prompt_for_string("TCP PORT of QNetd server", default=5403)
-    qdevice_algo = prompt_for_string("QNetd decision ALGORITHM (ffsplit/lms)", default="ffsplit")
-    qdevice_tie_breaker = prompt_for_string("QNetd TIE_BREAKER (lowest/highest/valid node id)", default="lowest")
-    qdevice_tls = prompt_for_string("Whether using TLS on QDevice/QNetd (on/off/required)", default="on")
-    qdevice_heuristics = prompt_for_string("Heuristics COMMAND to run with absolute path; For multiple commands, use \";\" to separate")
-    qdevice_heuristics_mode = prompt_for_string("MODE of operation of heuristics (on/sync/off)", default="sync") if qdevice_heuristics else None
+    while True:
+        try:
+            qdevice.QDevice.check_package_installed("corosync-qdevice")
+            break
+        except ValueError as err:
+            logger.error(err)
+            if confirm("Continue when this package installed?"):
```

Comment:
This message should be more explicit, for example,

```
Please install the package manually and press Enter to continue.
```

Reply:
Changed as
```
Please install the package manually and press 'y' to continue
```